catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegate.java

```diff
@@ -764,9 +764,7 @@ private FeatureIdType createFeatureIdFilter(final String id) {
   }
 
   private boolean isValidInputParameters(String propertyName, Object literal) {
-    return !(literal == null
-        || StringUtils.isEmpty(propertyName)
-        || StringUtils.isEmpty(literal.toString()));
+    return !(literal == null || StringUtils.isEmpty(propertyName));
```

Comment:
this was a small improvement I wanted to slip into this PR. Empty string values shouldn't be considered invalid in a query.

Reply:
Consider adding a unit test for this change.